### DISCUSSION TITLE
Save a website with expandable and collapsable elements

### DISCUSSION THREAD

**Discussion by christiandull** [gildas-lormeau/SingleFile#916] [General]
I want to save a website with expandable and collapsable elements ([https://www.vcrp-ecademy.de/module/online-kurse/faq-online-kurse/]).

The problem is, when i open the second element, the website itself is closing the first element. SingleFile is only saving the visible element. In the saved SingleFile file the other elements are not clickable.

For me there could be 3 solutions:

1. SingleFile saved files makes the elements also clickabel to open and show them
2. A option in SingleFile to expand all elements on a website or another extension that can do that (i have not found any)
3. A way to prevent the elements from closing when open the next one.

I hope you or someone can help me.

**Comment by gildas-lormeau** [comment]
This is a recurring issue. It is even documented a little bit in the FAQ here: https://github.com/gildas-lormeau/SingleFile/blob/master/faq.md#why-dont-interactive-elements-like-folding-titles-dynamic-maps-or-carousels-work-properly-in-saved-pages.

I was able to save the page you linked and keep the elements clickable by disabling 'HTML Content > save raw page' and disabling `Other resources > remove scripts`. However, images are not displayed properly.
Unfortunately, as you can see, this type of procedure is not reliable. Sometimes it works, sometimes not. Our only hope is that one day developers will realize that JavaScript is not necessary at all to implement collapsible elements.

I'm moving this issue to the Discussion tab to make it easily visible to everyone.

**Reply by christiandull** [reply]
Thanks for answer. With the 2 changed settings it works to keep them clickable.

That is great to know a solution that can work :)

**Comment by KnIfER** [comment]
users can use userscripts to fix. lol

**Reply by gildas-lormeau** [reply]
Feel free to contribute and explain better solutions, if you have any.
